services/bankid-api/lambdas/auth.js

```diff
@@ -1,7 +1,6 @@
 import to from 'await-to-js';
```

Comment:
How come you made changes to this function?

Reply:
Niko found it as was checking the error msgs when I was trying to  connect.